### DISCUSSION TITLE
[Help Wanted] Built-in Tag Folder Migration?

### DISCUSSION THREAD

**Discussion by scottbourne** [standardnotes/forum#2787] [General]
**Describe the issue**
With the new native, built-in tag folders announced, how do we migrate over to this functionality and disable and remove the extension?  What version does this go-live in?

A sequence of steps would be nice, so someone doesn't make a mess of things, or some words about how it's being done behind the scenes and there is nothing for the end user to do.  Just not at all clear how this is going to happen.

**Comment by atmoio** [comment]
You should see a "Migration Available" text if you're running the latest version, next to where it says "Tags" in your tags pane. Clicking that should give you a pretty good description of how it all works.

**Comment by scottbourne** [comment]
Do not see 'Migration Available' text.  I am running 3.11.2 on Ubuntu 20.04.4, which I think is the latest version.

**Comment by atmoio** [comment]
It only appears if you have any tags that contain a "." character. I believe it also checks if you are entitled to the Folders feature which requires a paid plan. Otherwise you can just start using folders by dragging and dropping tags onto one another.

**Comment by viharsheth** [comment]
Have a similar question. I'm a Pro user and have the Folders extension installed. If I uninstall it will I lose all of my tags?

**Comment by effieeee** [comment]
Hi @viharsheth, if you've been able to complete the migration to the new native Folders component, it should be safe to uninstall the old Folders extension 👍

**Comment by viharsheth** [comment]
How do I know if the migration has been done?

**Comment by effieeee** [comment]
You'll know it's done if the **Migration available** prompt is no longer there, and if you're now able to see the same folder hierarchies that you had with the old Folders extension! (the default icon for the native Folders component is the hash icon, but you can customize this in the right click context menu of a folder!)
